rotkehlchen/chain/ethereum/modules/aave/v2/accountant.py

```diff
@@ -94,7 +94,7 @@ def _process_withdraw(
                 taxable=True,
                 extra_data={'tx_hash': event.tx_hash.hex()},
             )
-            self.assets_borrowed[key] = ZERO
+            self.assets_supplied[key] = ZERO
```

Comment:
and this is bug 3.

All 3 known as copy pasta without bothering to test the code.